functional-tests/test-bed/src/main/java/org/arquillian/smart/testing/ftest/testbed/rules/TestBed.java

```diff
@@ -101,15 +106,15 @@ private void after() throws Exception {
     }
 
     private String targetRepoPerTestFolder(Description description) {
-        return GitClone.GIT_REPO_FOLDER
+        return gitClone.getGitRepoFolder()
```

Comment:
well inspecting the code I saw that this was an static field, so we can have problems when run in parallel threads.